tds-dll-mssql/src/main/java/tds/dll/mssql/StudentDLL.java

```diff
@@ -1758,6 +1758,23 @@ public boolean _AllowAnonymousTestee_FN (SQLConnection connection, String client
     return false;
   }
 
+  public boolean _SecureBrowserRequired_FN (SQLConnection connection, String clientname) throws ReturnStatusException {
```

Comment:
Why did you choose to add this within the TDS infrastructure rather than the microservices?  Is it because of how the loader system pulls all this data?